packages/sdk/src/job.ts

```diff
@@ -10,15 +10,16 @@ export class Job extends BaseApi {
    * Use PipcookClient instead.
    * @private
    */
-  constructor(url: string) {
-    super(`${url}/job`);
+  constructor(url: string, opts?: InitOption) {
+    super(`${url}/job`, opts);
   }
 
   /**
    * list all jobs.
    * @param filter the filter to list the jobs.
    * @returns The jobs list.
    */
+  @errorHandle()
```

Comment:
I think there is no need to use decorators, we should handle possible errors for all functions.

Reply:
You mean we should do try-catch in every function?

Reply:
Exactly, if someone specifies an error callback, he expects to handle all errors.